src/GE/tests/EmployeeTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: goran.erhartic
+ * Date: 28/3/2017
+ * Time: 9:04 AM
+ */
+
+use PHPUnit\Framework\TestCase;
+
+define('Employee', 'GE\Person\Employee');
+
```

Comment:
You don't want to use constants like this, they should be used only for config values. Use setup